doc/nrf/glossary.rst

```diff
@@ -126,6 +136,11 @@ Glossary
    Data Terminal Ready (DTR)
       A control signal in RS-232 serial communications transmitted from data terminal equipment, such as a computer, to data communication equipment.
 
+   Delivery Traffic Indication Message (DTIM)
+      A message in a beacon sent for every DTIM period.
+      This is a message present in the beacon frame.
+      It determines when the device is awake and asleep.
```

Comment:
Can we have something like below? 
Delivery Traffic Indication Message (DTIM) is a kind of TIM which informs the clients about the presence of buffered multicast/broadcast data on the access point. It is generated within the periodic beacon at a frequency specified by the DTIM Interval.